client/settings/fraud-protection/components/exceeds-dollar-amount-rule/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { sprintf, __ } from '@wordpress/i18n';
+
+const USDollarComponent = ( { level } ) => {
+	const isHighProtectionLevel = 'high' === level;
+
+	if ( isHighProtectionLevel ) {
+		return (
+			<li>
+				{ __( 'An order exceeds', 'woocommerce-payments' ) }{ ' ' }
+				<strong>{ __( '$1,000.00.', 'woocommerce-payments' ) }</strong>
+			</li>
+		);
+	}
+
+	return (
+		<li>
+			{ __( 'An order exceeds', 'woocommerce-payments' ) }{ ' ' }
```

Comment:
Are we able to use `interpolateComponents` for all of these strings so that there aren't translations needed for each of the short strings?

Reply:
Thanks for this suggestion! I was unaware of `interpolateComponents`. I think this would be a good use for it where we are simply adding bold or italics to the text. I'll look at getting this updates soon.

Reply:
Updated in [8008338](https://github.com/Automattic/woocommerce-payments/pull/5450/commits/8008338ba1612bf9b075251bd2f66598e992c5d9).